Pokedex_ios/Helpers/ConstantVariables.swift

```diff
@@ -31,6 +31,16 @@ class ConstantVariables {
         TypeElement(type: Generation(id: 17, name: "dark")),
         TypeElement(type: Generation(id: 18, name: "fairy"))
     ]
+    static let pokemonGenerations = [
+        GenerationAux(name: "Generation I", description: "generation-i"),
```

Comment:
why aux?

Reply:
because i only use it for the filter

Reply:
use a more descriptive name, for example GenerationForPopup (I know is too long but it makes it readable once you check variables)

Reply:
sure, that name could work

Reply:
done!